packages/components/filters/src/filter-menu/trigger-button/trigger-button.styles.ts

```diff
@@ -81,8 +80,12 @@ export const removeButton = css`
 `;
 
 export const label = css`
-  flex-grow: 0;
+  flex: 0 0 auto;
+  overflow: hidden;
+  text-overflow: ellipsis;
   cursor: inherit;
+  white-space: nowrap;
+  max-width: ${designTokens.constraint6};
```

Comment:
@FilPob this is my attempt to handle cases when the label for the filter is very long, it truncates the label after 284px so that there is always some room left for showing selected values in the button
![Screenshot 2024-10-17 at 8 56 45 AM](https://github.com/user-attachments/assets/9d1448b6-a51c-42a4-9916-fcd485fc78ac)

Reply:
Awesome, that looks good to me